module/userProjects/helper.js

```diff
@@ -940,7 +993,7 @@ module.exports = class UserProjectsHelper {
                 );
                 if (!project.length > 0) {
                     throw {
```

Comment:
@Prajwal17Tunerlabs same like _assessmentDetails also dont call survey Service copy that api here